### PR TITLE
[test] Make it possible to run tests from root directory

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: tests/test_adapters.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import nvdlib.query_selectors as selectors
 
 
-SAMPLE_CVE_PATH = 'data/cve-1.0-sample.json'
+SAMPLE_CVE_PATH = os.path.join(os.path.dirname(__file__), 'data/cve-1.0-sample.json')
```
Comment (tisnik) [inline tests/test_adapters.py]: I'd suggest to use pathlib.Path instead the clunky os.path.join()

File: tests/test_collection.py [hunk 2]
```diff
@@ -7,7 +8,7 @@
 from nvdlib.collection import Collection
 
 
-SAMPLE_CVE_PATH = 'data/cve-1.0-sample.json'
+SAMPLE_CVE_PATH = SAMPLE_CVE_PATH = os.path.join(os.path.dirname(__file__), 'data/cve-1.0-sample.json')
```
Comment (tisnik) [inline tests/test_collection.py]: ditto

File: tests/test_manager.py [hunk 3]
```diff
@@ -63,7 +64,7 @@ def test___init__(self):
 
     def test__parse_metadata(self):
         """Test JSONFeedMetadata `_parse_metadata` method."""
-        meta_file = 'data/nvdcve-1.0-sample.meta'
+        meta_file = os.path.join(os.path.dirname(__file__), 'data/nvdcve-1.0-sample.meta')
```
Comment (tisnik) [inline tests/test_manager.py]: again :D

Comment (msrb) [reply tests/test_manager.py]: Migrated to pathlib :wink:

**Comment by msrb** [comment]
Nah... tests are green, but there is a lot of pylint and docstyle issues (not related to this PR) in this repository. I am going merge this and open a separate PR.

**Pull request merged**